### DISCUSSION TITLE
Should the Text property of Option<TType> as string instead of TType?

### DISCUSSION THREAD

**Discussion by gitasaurus** [microsoft/fluentui-blazor#1874] [General]
Should the [Text property of Option\<TType\>](https://github.com/microsoft/fluentui-blazor/blob/f16c0dd64dedc08421f589b15708cca7af67d3e8/src/Core/Components/List/Option.cs#L7) be **string** instead of '**TType?**'?

It would be more flexible if the templated value type could be different than the display value type.  For example, using Value as an enum and Text as the translated value name.

**Comment by vnbaaij** [comment]
That would be a breaking change.

**Comment by gitasaurus** [comment]
Fair point, thank you for the response.
